packages/langchain/src/orchestration/client.test.ts

```diff
@@ -129,4 +158,93 @@ describe('orchestration service client', () => {
       'Request failed with status code 400'
     );
   }, 1000);
+
+  it('supports streaming responses', async () => {
+    mockStreamInferenceWithResilience();
+
+    const client = new OrchestrationClient(config);
+    const stream = await client.stream([]);
+    let iterations = 0;
+    const maxIterations = 2;
+    let intermediateChunk: AIMessageChunk | undefined;
+
+    for await (const chunk of stream) {
+      iterations++;
+      intermediateChunk = !intermediateChunk
+        ? chunk
+        : intermediateChunk.concat(chunk);
```

Comment:
[req] Just concat all chunk content and compare output for the sake of code simplicity.

Also I generally have the feeling that sometimes the code may work, just the logic is not clean. Is the above code not equivalent to

```ts
intermediateChunk = intermediateChunk
        ? intermediateChunk.concat(chunk)
        : chunk;
```

And are we not just checking the first chunk with content? Can't we check the content instead of using the maxIteration logic? (But still just loop over all chunks and compare to snapshot)